blottertrax/helper/repost_checker.py

```diff
@@ -0,0 +1,67 @@
+import sqlite3
+import os
+
+
+class RepostChecker:
+    cursor = None
+    sql = None
+
+    @classmethod
+    def __init__(self):
+        print("a")
+        self.sql = sqlite3.connect('{}/../database/repost.db'.format(os.path.dirname(os.path.dirname(os.path.realpath(__file__)))))

```

Comment:
Actually I think it might be better to merge this file with the existing `database.py` and use the same database file but just create new tables in there. Doesn't feel like this needs its own database file